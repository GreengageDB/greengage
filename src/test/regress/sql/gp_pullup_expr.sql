-- This case is based on the reproduce steps reported on https://github.com/GreengageDB/greengage/issues/15149

CREATE TABLE issue_15149(c1 varchar);

SELECT t.c1 FROM (SELECT DISTINCT ON (upper(c1)) c1 COLLATE "C" FROM issue_15149)t;

DROP TABLE issue_15149;
