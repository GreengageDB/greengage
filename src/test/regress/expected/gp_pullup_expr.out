-- This case is based on the reproduce steps reported on https://github.com/GreengageDB/greengage/issues/15149
CREATE TABLE issue_15149(c1 varchar);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'c1' as the Greengage Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
SELECT t.c1 FROM (SELECT DISTINCT ON (upper(c1)) c1 COLLATE "C" FROM issue_15149)t;
 c1 
----
(0 rows)

DROP TABLE issue_15149;
